src/gui/insEdit.cpp

```diff
@@ -2713,6 +2788,9 @@ void FurnaceGUI::drawInsEdit() {
           if (ins->type==DIV_INS_FDS) {
             volMax=32;
           }
+          if (ins->type==DIV_INS_ES5506) {
+            volMax=65535;
```

Comment:
Woooooooooooooow, OK, let me tell you.
Be careful with such super high volume ranges as I still support 32-bit and therefore any signed multiplication that involves more than 31 bits will overflow.

Reply:
Current behavior is now using log volume, and Already converted almost all volume related variables to unsigned int.